### ISSUE TITLE
Imaging Core: Integrate new reconstructions tools

### ISSUE THREAD

**Issue by DTasev** [mantidproject/mantid#19411]
Considered tools:
- Astra Toolbox (next) https://github.com/mantidproject/mantid/issues/19412
- MuhRec
- ~Savu~ (this tool was found to produce bad results and is low priority)

For each tool consider:
1. Functionality - Does it have anything desirable?  
2. API - Must be easily integrable with current scripts  
3. Speed - How fast is it? Does it run in parallel, or MPI? Can it handle large datasets (~20GB) quickly?  
4. CUDA - Does it make use of CUDA?  
5. MPI - Does it have API that allows external usage of the MPI functionality? E.g. something like MPI being an optional feature on each filter.

**Comment by DanNixon** [comment]
This has been superseded by: https://github.com/mantidproject/mantidimaging/issues/11
